Sources/MapboxCoreNavigation/RouterDelegate.swift

```diff
@@ -1,6 +1,27 @@
 import Foundation
 import CoreLocation
 import MapboxDirections
+import Turf
+
+/**
+ Configuration for offsetting first route maneuver during rerouting.
+ */
+public enum ReroutingManeuverOffset {
```

Comment:
“Offset” implies a simple magnitude in a certain direction. “Buffer” would be a more accurate way of describing this behavior if we can’t simplify it down to a single number. If we can simplify it, “buffer” would also be less ambiguous than “offset”, which is usually a lateral offset in map contexts.